package.json

```diff
@@ -33,12 +34,14 @@
   },
   "homepage": "https://github.com/mironov/react-redux-loading-bar",
   "peerDependencies": {
+    "@types/react": ">=0.14.55",
```

Comment:
I am wondering if this needs to be added as peerDependency. Isn't this only required for dev where it is also added as dependency?